### PR TITLE
Client Services Pages Importer

### PR DESCRIPTION
**Pull request by hannessolo**
Importer for #72 

- Before: https://main--asprey--hlxsites.hlx.page/gb/en
- After: https://main--asprey--hlxsites.hlx.page/drafts/hhertach/draft-import/craftsmanship
    - https://main--asprey--hlxsites.hlx.page/drafts/hhertach/draft-import/about
    - https://main--asprey--hlxsites.hlx.page/drafts/hhertach/draft-import/heritage-trophies
    - https://main--asprey--hlxsites.hlx.page/drafts/hhertach/draft-import/trophies-awards
    - https://main--asprey--hlxsites.hlx.page/drafts/hhertach/draft-import/find-a-boutique-japan
    - https://main--asprey--hlxsites.hlx.page/drafts/hhertach/draft-import/find-a-boutique
    - https://main--asprey--hlxsites.hlx.page/drafts/hhertach/draft-import/find-a-boutique-switzerland
    - https://main--asprey--hlxsites.hlx.page/drafts/hhertach/draft-import/find-a-boutique-usa

See list of pages

### CODE REVIEW

File: tools/importer/import.js [hunk 1]
```diff
@@ -22,6 +22,20 @@ function mapVariantsToStyles(classList, maps) {
   return styles;
 }
 
+function mapParentVariantsToStyles(element, maps) {
```
Comment (jckautzmann) [inline tools/importer/import.js:25]: Could you please add some docs to explain what this function is doing? Maybe also add an example?

File: tools/importer/import.js [hunk 2]
```diff
@@ -128,6 +152,31 @@ function transformTitle(document) {
 }
 
 function transformHeroImage(document) {
+  document.querySelectorAll('.image').forEach((heroimage) => {
```
Comment (jckautzmann) [inline tools/importer/import.js:155]: I'd rename it `image` because at this point we don't know if it is a hero image.

File: tools/importer/import.js [hunk 3]
```diff
@@ -136,21 +185,202 @@ function transformHeroImage(document) {
...
+    const cells = [
+      ['Subnav'],
+      [[ul]],
+    ];
+    const table = WebImporter.DOMUtils.createTable(cells, document);
+
+    const metadataCells = [
+      ['Section Metadata'],
+      ['Style', 'text align center'],
+      ['Background', 'gray'],
```
Comment (jckautzmann) [inline tools/importer/import.js:224]: Shouldn't it be `grey`?

Comment (hannessolo) [reply tools/importer/import.js:224]: If you ask me (British English), yes it should be `grey`. But almost every css file I've seen so far working on projects used US style `gray` 🙂

Also I have to stop myself from writing `colour` instead of `color`.

File: tools/importer/import.js [hunk 4]
```diff
@@ -136,21 +185,202 @@ function transformHeroImage(document) {
...
+  });
+}
+
 function transformTeaser(document) {
   document.querySelectorAll('.teaser').forEach((teaser) => {
-    const styles = mapVariantsToStyles(teaser.classList, [
-      ['backgroundDeepPurple', 'backgroundDeepPurple'],
+    const classStyles = mapVariantsToStyles(teaser.classList, [
       ['imageTop', 'imageTop'],
+      ['backgroundOffWhite', 'background gray'],
```
Comment (jckautzmann) [inline tools/importer/import.js:373]: typo? `background grey`?

Comment (hannessolo) [reply tools/importer/import.js:373]: see https://github.com/hlxsites/asprey/pull/97#discussion_r1338319381 :)

File: tools/importer/import.js (hunk 3 above)
Comment (jckautzmann) [reply tools/importer/import.js:224]: ah right, I'm not used to the US spelling. Let's not start a US/UK war on spelling 🙂

File: tools/importer/import.js [hunk 5]
```diff
@@ -12,6 +12,42 @@
 /* global WebImporter */
 /* eslint-disable no-console, class-methods-use-this */
 
+function readContentUntil(startEl, stopCriteria) {
```
Comment (jckautzmann) [inline tools/importer/import.js:15]: it's hard to follow what this function is doing. I'd add some docs with an example.

File: tools/importer/import.js [hunk 6]
```diff
@@ -12,6 +12,42 @@
...
+      next = next.nextSibling || next.parentNode.nextSibling;
+    }
+  }
+  if (foundEnd) {
+    return content;
+  }
+  return null;
+}
+
+function removeEmptySections(main) {
```
Comment (jckautzmann) [inline tools/importer/import.js:40]: Isn't this function removing duplicate hr? If so, I'd call it `removeDuplicateHr`

**Comment by jckautzmann** [comment]
The importer code looks good to me. Now to be confident that it maps to the correct blocks needs to be tested/verified on real pages :)

**Review by Kronnox — APPROVED** [review]
